src/main/webapp/app/entities/participation-submission/participation-submission.component.html

```diff
@@ -0,0 +1,43 @@
+<h1>Submissions for Participation with id {{ participationId }}</h1>
+<br />
+<div class="table-responsive" *ngIf="submissions">
```

Comment:
It would be great if we could use the ngx-swimlane datatable here, because it can handle large datasets (we can have 1000+ data entries here).
There are good examples in their docs and the api should be self explanatory: https://github.com/swimlane/ngx-datatable

We already have one table that is implemented in this library: src/main/webapp/app/entities/programming-exercise/test-cases/programming-exercise-manage-test-cases.component.html

Reply:
I used ngx-datatable like in the programming-exercise-manage-test-cases.component.html and adapted the client side test cases :)